stories/cosmoz-autocomplete.stories.js

```diff
@@ -82,9 +82,44 @@ const css = html`
 			.textProperty=${'text'}
 			.value=${colors[0]}
 		></cosmoz-autocomplete>
-	`;
+	`,
+	contourStyling = () => html` <style>
+			cosmoz-autocomplete {
+				--cosmoz-input-border-radius: 4px;
+				--cosmoz-input-padding: 8px 0;
+				--cosmoz-input-label-width: auto;
+				--cosmoz-input-no-placeholder-label-bg: white;
+				--cosmoz-input-label-padding: 0;
+				--cosmoz-input-line-display: none;
+				--cosmoz-input-contour-size: 1px;
+				--cosmoz-input-background: white;
+				--cosmoz-autocomplete-chip-border-radius: 4px;
+			}
+			cosmoz-autocomplete::part(input-control) {
+				margin: 0 8px;
+			}
+			cosmoz-autocomplete::part(chip-clear) {
+				margin-left: 4px;
+			}
```

Comment:
I think you should expose these as css variables: `--cosmoz-autocomplete-input-control-margin` and `--cosmoz-autocomplete-chip-clear-margin`, instead of relying on parts. Parts are good, but css variables are better, because they cascade.